message_ix/model/MESSAGE/model_core.gms

```diff
@@ -248,7 +248,9 @@ Equations
     COMMODITY_BALANCE               commodity supply-demand balance constraint
     STOCKS_BALANCE                  commodity inter-temporal balance of stocks
     CAPACITY_CONSTRAINT             capacity constraint for technology (by sub-annual time slice)
-    CAPACITY_MAINTENANCE            constraint for technology capacity maintainance
+    CAPACITY_MAINTENANCE_HIST       constraint for capactiy maintainance  historical installation (built before start of model horizon)
+    CAPACITY_MAINTENANCE_CURR       constraint for capactiy maintainance of new capacity built in the current period (vintage == year)
+    CAPACITY_MAINTENANCE            constraint for capacity maintainance (at the end of previous periode)
```

Comment:
there is a typo here ('periode'), and I would rather call that text "constraint for capacity maintainance over the technical lifetime"